src/Components/UrlButtons.js

```diff
@@ -1,18 +1,24 @@
 import Box from "@mui/material/Box";
 import IconButton from "@mui/material/IconButton";
 import Typography from "@mui/material/Typography";
-import { Fragment } from "react";
 import crunchyroll from "../Styles/images/crunchyroll.png";
+import disneyPlus from "../Styles/images/disney_plus.png";
 import funimation from "../Styles/images/funimation.png";
+import google from "../Styles/images/google.png";
+import hboMax from "../Styles/images/hbo_max.png";
+import hulu from "../Styles/images/hulu.png";
 import netflix from "../Styles/images/Netflix.png";
+import prime from "../Styles/images/prime.png";
 
 export default function UrlButtons({ anime }) {
-  const urls = anime.urls.filter(
-    (url) =>
-      url.includes("crunchyroll") ||
-      url.includes("funimation") ||
-      url.includes("netflix")
-  );
+  const buttonData = getButtonData(anime.urls);
+
+  // Add a simple google link if there are none.
+  if (!buttonData.length) {
+    const encodedName = encodeURIComponent(anime.display_name);
+    const googleUrl = `https://www.google.com/search?q=${encodedName}`;
+    buttonData.push({ url: googleUrl, title: "Google It", image: google });
+  }
 
   return (
     <Box
```

Comment:
Not sure if you're still cooking on this shelf, but currently it gunks up our mobile view if there's 4+ urlButtons :(

Just adding a `flexWrap: "wrap"` here does a good enough job (if you're not already implementing buttons!)

Reply:
Oh, snap!  Nice catch -- I didn't even check that 😧.  Fix applied.